crates/rolldown/tests/fixtures/deconflict/issue_364/artifacts.snap

```diff
@@ -11,13 +11,13 @@ input_file: crates/rolldown/tests/fixtures/deconflict/issue_364
 import { default as assert } from "node:assert";
 
 //#region shared.js
-const a$2 = 'shared.js';
+const a$1$1 = 'shared.js';
```

Comment:
should be `a$n` rather than `a$1$1`

Reply:
Alright, I got you, let me test if performance got regressed